test/language/expressions/assignment/dstr/array-elem-put-obj-literal-prop-ref-init.js

```diff
@@ -0,0 +1,69 @@
+// This file was procedurally generated from the following sources:
+// - src/dstr-assignment/array-elem-put-obj-literal-prop-ref-init.case
+// - src/dstr-assignment/default/assignment-expr.template
+/*---
+description: The DestructuringAssignmentTarget of an AssignmentElement can extend to LHSExpressions if it is neither an ObjectLiteral nor an ArrayLiteral and its AssignmentTargetTyp is simple. Using MemberExpression (ObjLiteral + identifier) with initializer. (AssignmentExpression)
+esid: sec-variable-statement-runtime-semantics-evaluation
+es6id: 13.3.2.4
```

Comment:
Get rid of these please

Reply:
these are originally from the template files, which I prefer to do in another PR, replacing them with the proper esid's if necessary.

Reply:
in any case, these are valid ES6 codes.

Reply:
The new commits address only the templates used in here.